tests/interchain/provider_consumers_suite.go

```diff
@@ -0,0 +1,98 @@
+package interchain
+
+import (
+	"context"
+	"cosmos/interchain-security/tests/interchain/chainsuite"
+	"strconv"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
+	"github.com/strangelove-ventures/interchaintest/v8/chain/cosmos"
+	"github.com/strangelove-ventures/interchaintest/v8/ibc"
+	"github.com/stretchr/testify/suite"
+)
+
+type ProviderConsumersSuite struct {
```

Comment:
Nit: Consider calling this `ConsumerChangeoverSuite` and rename the files to match.

Reply:
this suite contains test for changeover from sovereign to consumer but also has another chain that is consumer from the beginning and test reward distribution for that other chain